doc/releases/reef.rst

```diff
@@ -0,0 +1,219 @@
+====
+Reef
+====
+
+Reef is the 18th stable release of Ceph. It is named after the reef squid
+(Sepioteuthis).
+
+v18.2.0 Reef
+============
+
+This is the first stable release of Ceph Reef.
+
+CephFS
+~~~~~~
+
+* CephFS: The ``mds_max_retries_on_remount_failure`` option has been renamed to
+  ``client_max_retries_on_remount_failure`` and moved from ``mds.yaml.in`` to
+  ``mds-client.yaml.in``. This change was made because the option has always
+  been used only by the MDS client.
+* CephFS: It is now possible to delete the recovered files in the
+  ``lost+found`` directory after a CephFS post has been recovered in accordance
+  with disaster recovery procedures.
+
+MGR
+~~~
+
+* mgr/snap_schedule: The snap-schedule manager module now retains one snapshot
+  less than the number mentioned against the config option
+  ``mds_max_snaps_per_dir``. This means that a new snapshot can be created and
+  retained during the next schedule run.
+* The ``ceph mgr dump`` command now outputs ``last_failure_osd_epoch`` and
+  ``active_clients`` fields at the top level. Previously, these fields were
+  output under the ``always_on_modules`` field.
+  
+RADOS
+~~~~~
+
+* The trimming of PGLog dups is now controlled by the size instead of the
+  version. This fixes the PGLog inflation issue that occurred when the online
+  (in OSD) trimming got jammed after a PG-split operation. A new offline
+  mechanism has been added: the ``ceph-objectstore-tool`` has added a
+  ``trim-pg-log-dups`` operation that targets situations where an OSD is unable
+  to boot due to inflated dups. When this is the case, the ``You can be hit by
+  THE DUPS BUG`` warning will be visible in the OSD logs. Relevant tracker:
+  https://tracker.ceph.com/issues/53729
```

Comment:
@zdover23 - Edited/reviewed version from 2023 June 18 (with extraneous hyphens removed): Trimming of PGLog dups is now controlled by the size rather than the version. This change fixes the PGLog inflation issue that was happening when the online (in OSD) trimming got jammed after a PG split operation. Also, a new offline mechanism has been added: ``ceph-objectstore-tool`` has a new operation called ``trim-pg-log-dups`` that targets situations in which an OSD is unable to boot because of the inflated dups. In such situations, the "You can be hit by THE DUPS BUG" warning is visible in OSD logs. Relevant tracker: https://tracker.ceph.com/issues/53729

Reply:
Accepted.